### DISCUSSION TITLE
Completely disable sleep?

### DISCUSSION THREAD

**Discussion by iforce2d** [Ralim/IronOS#1665] [Q&A] [answered]
Hello, I'm making a little automatic soldering machine which will make very small slow movements over a long time period, and I would like the iron to never go into sleep mode. I see there is a setting for motion sensitivity where "0 == motion detection turned off". Does this mean that the heat will stay on all the time?

**Comment by discip** [comment, marked as answer]
@iforce2d
If you set **`Motion sensitivity`** to **`0`** sleep is disabled.
